tests/optional_chaining/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -0,0 +1,36 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`chaining.js 1`] = `
+var street = user.address?.street
+var fooValue = myForm.querySelector('input[name=foo]')?.value
+
+obj?.prop;
+obj?.[expr];
+func?.(...args);
+
+a?.();
```

Comment:
Coffeescript somehow made it work :)

![img_0790](https://user-images.githubusercontent.com/197597/28807819-d74aaf22-7634-11e7-8b7b-500dc122cca5.PNG)